components/builder-api/habitat/hooks/run

```diff
@@ -4,6 +4,10 @@ export HOME="{{pkg.svc_data_path}}"
 export RUST_LOG="{{cfg.log_level}}"
 export RUST_BACKTRACE=1
 
+title="Service Start"
```

Comment:
Should we also add a post-run hook to indicate that builder-api has started?

Reply:
I don't think we strictly need it right now.. eg, if the service is thrashing, we'll see a spike of service starts.  But we can certainly consider adding more events in hooks once we're happy with the shape of this one.